redbot/cogs/mod/kickban.py

```diff
@@ -334,6 +334,15 @@ def remove_processed(ids):
                         user_id=user_id
                     )
 
+        member = namedtuple("Member", "id guild")
+        async with self.settings.guild(guild).current_tempbans() as guild_tempbans:
+            for uid in guild_tempbans.copy():
+                for user_id in user_ids.copy:
```

Comment:
Im not entirely certain what you are trying to do here and in line 341 and 342.

also `.copy` isnt a properpy nor attribute of list did you mean `.copy()` ? if so why are you wanting to use a copy here?